src/type-checker/type-checker.ts

```diff
@@ -108,8 +108,7 @@ export class TypeChecker {
     program: ts.Program
   ): ts.Node | undefined {
     try {
-      // @ts-ignore Internal method
-      return ts.getTouchingPropertyName(
+      return (ts as any).getTouchingPropertyName(
```

Comment:
Hmm I'd rather keep a `@ts-expect-error` here, explaining it's an internal method we are using. I don't like parsing stuff to `as any` because it bypasses type checking. `@ts-expect-error` leaves the door open for the problem to be solved in the future.